### PR TITLE
fix(@desktop): enable the hover effects by default

### LINKED ISSUE #6254: There is no hint for the Chat tab
# Bug Report

## Steps to reproduce
1. Sign in and open Community Portal or Settings
2. Hover over the Chat icon in the navigation left bar

#### Expected behavior
The hint 'Chat' is shown
#### Actual behavior
There is no hint for the Chat tab
### Additional Information
image

Status desktop version: https://ci.status.im/job/status-desktop/job/branches/job/macos/job/master/2246/
Operating System: macOS Monterey 12.3 Beta (21E5227a)

### PR DESCRIPTION
**Pull request by caybro**
Close #6254

- since Qt 5.xy, hover is not enabled by default for QQC2, so enable it
unconditionally as we are a desktop app anyway
- this fixes several hover effects being broken, mostly for builtin
components like MenuItem and some buttons (eg. the leftmost NavBar)
where we haven't enabled those with `hoverEnabled: true` explicitely
- additionally, provide a tooltip for the default "chat" section which
doesn't seem to supply its own (unlike the other sections)

### What does the PR do

Fixes mouse over (hover) effects being (mostly) broken

### Affected areas

All areas of status-desktop

### Screenshot of functionality
Snímek obrazovky z 2022-07-04 12-41-28
Snímek obrazovky z 2022-07-04 12-42-10

### CODE REVIEW

**Review by PascalPrecht — CHANGES_REQUESTED** [review]
The actual fix is of this commit is awesome!
Didn't know this existed. Setting `hoverEnabled` explicitly is quite annoying.

Bonus points if the fix for the tooltip is not in the commit that fixes the hover (no biggie if you don't feel like it)

File: ui/app/mainui/AppMain.qml [hunk 1]
```diff
@@ -277,7 +277,9 @@ Item {
                 name: model.icon.length > 0? "" : model.name
                 icon.name: model.icon
                 icon.source: model.image
-                tooltip.text: model.name
+                tooltip.text: model.name ||
+                              //% "Chat"
+                              qsTrId("chat")
```
[the remark this answers is not in the document]
Comment (caybro) [reply ui/app/mainui/AppMain.qml:282]: Alright, sure will fix it. It's just that the existing code still contains a _lot_ of `qsTrid()` instances... worth cleaning that up at once?

**Review by osmaczko — APPROVED** [review]
:+1:

File: ui/app/mainui/AppMain.qml (hunk 1 above)
[the remark this answers is not in the document]
Comment (caybro) [reply ui/app/mainui/AppMain.qml:282]: It's just empty, dunno why :/ Not enough nim-fu to find the correct cause though

Will split this into 2 commits, no prob

[the remark this answers is not in the document]
Comment (osmaczko) [reply ui/app/mainui/AppMain.qml:282]: > Alright, sure will fix it. It's just that the existing code still contains a _lot_ of `qsTrid()` instances... worth cleaning that up at once?

https://github.com/status-im/status-desktop/pull/5327 will be merged after the release

[the remark this answers is not in the document]
Comment (caybro) [reply ui/app/mainui/AppMain.qml]: @PascalPrecht fixed and split into 2 commits

[the remark this answers is not in the document]
Comment (caybro) [reply ui/app/mainui/AppMain.qml]: @osmaczko fixed (using qsTr())

**Review by alaibe — APPROVED** [review]

**Review by jrainville — CHANGES_REQUESTED** [review]
Nice fixes. The fix for the tooltip should be done in Nim however. I don't blame you for not knowing, it's not clear

File: ui/app/mainui/AppMain.qml [hunk 2]
```diff
@@ -277,7 +277,7 @@ Item {
                 name: model.icon.length > 0? "" : model.name
                 icon.name: model.icon
                 icon.source: model.image
-                tooltip.text: model.name
+                tooltip.text: model.name || qsTr("Chat")
```
[the remark this answers is not in the document]
Comment (caybro) [reply ui/app/mainui/AppMain.qml]: Alright, fixed and (still) works! :)

**Review by jrainville — APPROVED** [review]
Good job

**Review by PascalPrecht — APPROVED** [review]
Great job

**Comment by caybro** [comment]
@noelia-santos can you please give this one a test too please? TIA

**Review by Hbouaz — APPROVED** [review]
nice one

**Pull request merged**